src/ray/gcs/tables.cc

```diff
@@ -231,7 +236,8 @@ template <typename ID, typename Data>
 Status Table<ID, Data>::Add(const DriverID &driver_id, const ID &id,
                             std::shared_ptr<DataT> &dataT, const WriteCallback &done) {
   num_adds_++;
-  auto callback = [this, id, dataT, done](const std::string &data) {
+  auto callback = [this, id, dataT, done](const CallbackReply &reply) {
+    RAY_IGNORE_EXPR(reply);
```

Comment:
Did the compiler complain any error without adding `RAY_IGNORE_EXPR`?
I think it should be fine. Because `reply` is a function parameter, not a local variable.